libtransmission/verify.cc

```diff
@@ -56,7 +56,7 @@ int tr_verify_worker::Node::compare(tr_verify_worker::Node const& that) const
     return 0;
 }
 
-bool tr_verify_worker::verifyTorrent(tr_torrent* tor, bool const* stop_flag)
+bool tr_verify_worker::verifyTorrent(tr_torrent* tor, std::atomic<bool>& stop_flag)
```

Comment:
nit: can be `const` qualified?

Reply:
Fixed in 554ba06ae2f1aa351d9a7da44559e7e312a39366 :smile_cat: